src/sequencer-predict.sh

```diff
@@ -113,7 +113,7 @@ fi
 echo
 
 echo "Generating predictions"
-python3 $CURRENT_DIR/lib/OpenNMT-py/translate.py -model $ROOT_DIR/model/model.pt -src $CURRENT_DIR/tmp/${BUGGY_FILE_BASENAME}_abstract_tokenized_truncated.txt -output $CURRENT_DIR/tmp/predictions.txt -beam_size $BEAM_SIZE -n_best $BEAM_SIZE &>/dev/null
+python3 $CURRENT_DIR/lib/OpenNMT-py/translate.py -model $ROOT_DIR/model/model.pt -src $CURRENT_DIR/tmp/${BUGGY_FILE_BASENAME}_abstract_tokenized_truncated.txt -output $CURRENT_DIR/tmp/predictions.txt -beam_size $BEAM_SIZE -n_best $BEAM_SIZE 1>/dev/null
```

Comment:
WDYT about verbose mode (-v option to output log from translate.py)? I can see that the log from OpenNMT to be useful for debugging, but sometimes I found it to be annoying when the beam size is large.

Reply:
If we use `1>/dev/null` we only hide stdout, so it will only print errors, if there are any.

Of course we can also make verbose mode and print everything.